src/Microsoft.DotNet.Wpf/src/PresentationFramework/System/Windows/Documents/glyphs.cs

```diff
@@ -406,14 +406,14 @@ private static bool IsEmpty(string s)
         /// <param name="glyphIndex"></param>
         /// <returns>true if glyph index is present, false if glyph index is not present.</returns>
         private bool ReadGlyphIndex(
-            string      valueSpec,
+            ReadOnlySpan<char> valueSpec,
```

Comment:
I think we should keep the previous format (Argument name in a column to keep the same format in the file).

Reply:
Done